appintro/src/main/java/com/github/paolorotolo/appintro/AppIntro2.kt

```diff
@@ -1,22 +1,35 @@
 package com.github.paolorotolo.appintro
 
+import android.annotation.TargetApi
 import android.graphics.drawable.Drawable
+import android.os.Build
 import android.os.Bundle
 import android.view.View
 import android.widget.ImageButton
 import androidx.annotation.ColorInt
+import androidx.annotation.IdRes
 import androidx.constraintlayout.widget.ConstraintLayout
 
 abstract class AppIntro2 : AppIntroBase() {
 
-    var backgroundView: Int? = null
+    @IdRes
+    var backgroundResource: Int? = null
         set(value) {
             field = value
             if (field != null) {
                 backgroundFrame.setBackgroundResource(field!!)
             }
         }
 
+    var backgroundDrawable: Drawable? = null
+        @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
Replace with an If-then-else on the API level pls

Reply:
Done😄